TeamMemberFiles/joseph.allen.txt

```diff
@@ -0,0 +1,5 @@
+Joseph Allen
+Please call me Joe
+I'm in 294
+I prefer Microsoft, Java MySQL
+I work full time, 12h shifts rotating days/nights every two weeks
```

Comment:
Awesome work Joe!
Please clean up your side of the equation to protect your information